### DISCUSSION TITLE
Async is dead. Long live Async.

### DISCUSSION THREAD

**Discussion by ioquatix** [socketry/async#135] [General]
## What's Changed
* Take advantage of Ruby 3 scheduler. by @ioquatix in https://github.com/socketry/async/pull/111

**Full Changelog**: https://github.com/socketry/async/compare/v1.29.1...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/socketry/async/releases/tag/v2.0.0'>Async is dead. Long live Async.</a>.</em>

**Comment by tan-wei** [comment]
`async` is great! Why *Async is dead*?

**Reply by olleolleolle** [reply]
The team wants to promise continuity, compare https://en.wikipedia.org/wiki/The_king_is_dead,_long_live_the_king!

**Reply by tan-wei** [reply]
@olleolleolle LOL, thanks.
